contracts/Reputation.sol

```diff
@@ -0,0 +1,111 @@
+/*
+
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+
+*/
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solium-disable linebreak-style
+
+pragma solidity 0.8.6;
+
+import "openzeppelin-contracts-sol8/token/ERC20/ERC20.sol";
+import "openzeppelin-contracts-sol8/access/Ownable.sol";
+
+contract Reputation is ERC20, Ownable {
+
+    modifier validInput(
+        address[] memory _repRecipients,
+        uint[] memory _repAmounts
+    ){
+        require(
+            _repRecipients.length == _repAmounts.length,
+            "Reputation: number of reputation holders doesn't match number of reputation amounts"
+        );
+        require(
+            _repRecipients.length <= 200,
+            "Reputation: maximum number of reputation holders and amounts of 200 was exceeded"
+        );
+        _;
+    }
+
+    constructor(
+        address[] memory _repRecipients,
+        uint256[] memory _repAmounts
+    )
+    ERC20("PrimeDAO Reputation", "REP")
+    {
+        _batchMint(_repRecipients, _repAmounts);
+    }
+
+    function transfer(
+        address recipient,
+        uint256 amount
+    ) public override pure returns(bool) {
+        return false;
+    }
+
+    function transferFrom(
```

Comment:
For my understanding. Did you override the transfer/transferFrom function so you effectively disabled the function from  the original ERC20 contract?

Reply:
@kinspotting did that with exactly this intention :)

Reply:
Great then I understood what was happening. Thanks